server/internal/graphql/resolver/operator.go

```diff
@@ -86,7 +86,20 @@ func (r *Resolver) changeOperatorStatus(ctx context.Context, provider models.Pro
 			r.Log.Info("skipping operator deployment (in disabled mode)")
 			return
 		}
-		op, _ := ctx.Value(models.MesheryControllerHandlersKey).(map[string]map[models.MesheryController]controllers.IMesheryController)
+		op, exists := ctx.Value(models.MesheryControllerHandlersKey).(map[string]map[models.MesheryController]controllers.IMesheryController)
+		if !exists {
+			r.Log.Error(err)
```

Comment:
Is the error here always Meshkit error?

Reply:
Requested changes from last week are pending consideration, @Azanul